### ISSUE TITLE
Use Github's issue templates to better triage issues

### ISSUE THREAD

**Issue by rlouf** [aesara-devs/aesara#1310]
The [new issue template system](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) allows to define compulsory fields when filing an issue. We can also use the template to [redirect users](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) when their request is not an issue (feature request, question about Aesara, etc).

**Comment by brandonwillard** [comment]
Yeah, we definitely need to do this, because it looks like it will produce instruction messages for issue creators/submitters that will not be included in the resulting issues.
